src/priyanka/BankingSystem.java

```diff
@@ -0,0 +1,87 @@
+package priyanka;
+
+public class BankingSystem {
+
+	int balance;
+	static int totalCredit;
+	static int totalDebit;
+	static int totalPrintBalance;
+	int credit;
+	int debit;
+	int printBal;
+
+	void accountHolder(String accountHolderName) {
+		if (accountHolderName == "Priyanka") {
```

Comment:
Here this looks wrong what is user3 wants to do some operation on Bank, You not handle this scenario.

rest code is excepted,
refer below code:
https://github.com/KrishnaKTechnocredits/JAVATechnoOct2020/pull/191/files